ui/app/styles/clinical/_observations.scss

```diff
@@ -428,6 +428,11 @@ concept-set {
     padding-bottom: 30px;
     display: inline-block;
   }
+  table {
```

Comment:
this will apply to all table elements, wouldn't it? why not use a class and apply it at appropriate place?